Tests/SonarScanner.MSBuild.Tasks.UnitTests/GetAnalyzerSettingsTests.cs

```diff
@@ -223,42 +223,58 @@ public void GetAnalyzerSettings_ConfigExists_NewBehaviour_SettingsMerged()
         public void ShouldMerge_OldServerVersion_ReturnsFalse()
         {
             // The "importAllValue" setting should be ignored for old server versions
-            CheckShouldMerge("7.3.1", "cs", importAllValue: "true", expected: false);
-            CheckShouldMerge("6.7.0", "vbnet", importAllValue: "true", expected: false);
+            var logger = CheckShouldMerge("7.3.1", "cs", importAllValue: "true", expected: false);
```

Comment:
I was doing some testing this morning and couldn't work out why external issues weren't being uploaded, so I've added some more logging to the task.